scripts/setup.sh

```diff
@@ -56,19 +56,48 @@ setup_haskell() {
   ln -sf ${STACK_BIN_PATH} ${HVN_DEST}/.stack-bin
 
   msg "Installing helper binaries..."
-  local STACK_LIST="ghc-mod hlint hasktags hscope pointfree pointful hoogle apply-refact machines-directory-0.2.0.9 machines-io-0.2.0.13 codex-0.5.0.2"
-  stack --resolver ${STACK_RESOLVER} install ${STACK_LIST} --verbosity warning ; RETCODE=$?
-  [ ${RETCODE} -ne 0 ] && exit_err "Binary installation failed with error ${RETCODE}."
-
-  # Install hindent 5 via pinned nightly
-  stack --resolver nightly-2016-12-28 install hindent --verbosity warning ; RETCODE=$?
-  [ ${RETCODE} -ne 0 ] && exit_err "hindent installation failed with error ${RETCODE}."
+  # Install ghc-mod via active stack resolver for maximum out-of-the-box compatibility.
+  stack --resolver ${STACK_RESOLVER} install ghc-mod --verbosity warning ; RETCODE=$?
+  [ ${RETCODE} -ne 0 ] && exit_err "Installing ghc-mod failed with error ${RETCODE}."
+
+  # Install hindent via pinned LTS to ensure we have version 5.
```

Comment:
I think you're right about this one...I tried adding `hindent >= 5.0` to the dependencies cabal file and that breaks the solver. Guess we have to do it separately like you did.